src/s2mMuscleFatigueDynamicState.cpp

```diff
@@ -41,6 +41,24 @@ double s2mMuscleFatigueDynamicState::previousRestingFibers() const {
     return m_previousRestingFibers;
 }
 
+s2mVector s2mMuscleFatigueDynamicState::getTimeDerivativeState() const
```

Comment:
As previously discussed, I do not want to extract data using an s2mVector since there is no logical way it can be done. They should be individually harvested by the user from their respective functions.